flink-runtime/src/main/java/org/apache/flink/runtime/jobmaster/JobManagerServices.java

```diff
@@ -116,12 +116,17 @@ public static JobManagerServices fromConfiguration(
 		final FiniteDuration timeout;
 		try {
 			timeout = AkkaUtils.getTimeout(config);
-		} catch (NumberFormatException e) {
+		}
+		catch (NumberFormatException e) {
```

Comment:
IMO, we should have a discuss about whether needing line break behand `}`. enable it to checkstyle